src/components/dataset/introduction.vue

```diff
@@ -51,8 +56,43 @@ export default {
 </script>
 
 <style lang="scss">
+@import '../../assets/scss/_variables.scss';
+
 #dataset-introduction {
-  .step-number { font-style: italic; }
+  &.panel-dialog {
+    width: 100%;
+    max-width: 770px;
+    margin-bottom: 20px;
+
+    .panel-title {
+      font-size: 30px;
+    }
+  }
+
+  .step-number {
+    font-style: italic;
+  }
+
+  .image-wrapper img {
+    width: 100%;
+    max-width: 720px;
+  }
+}
+
+@media (min-width: $screen-md-min) {
+  #dataset-introduction {
+    &.panel-dialog {
+      max-width: 1045px;
+    }
+
+    .row {
+      display: flex;
```

Comment:
Why is this needed? Isn't flexbox more of an alternative to the Bootstrap grid? (I actually like flexbox a lot and often reach for it over Boostrap.)

Reply:
I used it to vertically center the image on some screen widths